tests/dragonfly/replication_test.py

```diff
@@ -53,14 +53,14 @@ async def wait_for_replicas_state(*clients, state="stable_sync", timeout=0.05):
         pytest.param(8, [8, 8], dict(key_target=1_000_000, units=16), 50_000, marks=M_STRESS),
     ],
 )
+@pytest.mark.parametrize("mode", [({}), ({"cache_mode": "true"})])
```

Comment:
Hmm 🤔 Do we really want to run every test scenario twice... Maybe 🤷🏻

Reply:
That was the point yes, to see how cache mode behaves on all the variations. We got a bug somewhere(as mentioned aboce), I won't merge this yet